src/analyses/apron/relationAnalysis.apron.ml

```diff
@@ -44,10 +44,10 @@ struct
     if ContextUtil.should_keep ~isAttr:GobContext ~keepOption:"ana.relation.context" ~removeAttr:"relation.no-context" ~keepAttr:"relation.context" fd then
       x
     else
-      D.bot () (* just like startstate, heterogeneous RD.bot () means top over empty set of variables *)
+      D.top ()
```

Comment:
Why does this matter at all? If we're not keeping contexts, then either should be fine. This is just used for the constraint variable and no lattice operations are done on it.

Reply:
It doesn't. I think we just changed it to be consistent with the comment.

Reply:
And personally I think a `T` context is less surprising than a $$\bot$$  to people new to the framework. If we didn't have the side-effects and partial contexts, this would be related to the initial value for which $$\bot$$ is unsound.

Reply:
Makes sense. Since `bot` was top on empty environment, that's essentially the meaning it already had, just in a roundabout way.